### DISCUSSION TITLE
WhatsApp Report and Block

### DISCUSSION THREAD

**Discussion by zyereon** [WhiskeySockets/Baileys#378] [New WhatsApp Features]
I want to block and report a number with baileys, but baileys only have the feature of block and unblock numbers and not block and report

**Comment by Kaiomkky** [comment]
Up

**Comment by Kaiomkky** [comment]
Up

**Comment by shizothetechie** [comment]
> I want to block and report a number with baileys, but baileys only have the feature of block and unblock numbers and not block and report

Good idea, Baileys must be updated with report user features 😀
